privacy_gdpr/wizards/res_partner_gdpr.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import base64
+from openerp import api, fields, models
+
+FIELDS_GDPR = [
+    'image',
+    'name',
+    'street',
+    'street2',
+    'zip',
+    'city',
+    'website',
+    'function',
+    'phone',
+    'mobile',
+    'fax',
+    'email',
+    'title',
+    'child_ids',
```

Comment:
@hbrunn They are configurable via the wizard, these are just some sane default fields that will be set when you open the wizard, you are free to remove them from the `many2many`.

Reply:
...every single time I gdpr-remove a person?

Reply:
You should define them on a paramater or a function that could be edited something like ::

    @api.model
    def _gdpr_remove_fields(self):
        return ['street, ....]

Reply:
@hbrunn Yes, now of course after the latest changes (which you can review) you can select multiple customers and click the wizard, these fields will be automatically added for you because these are probably the ones you want to remove (suggestion by @lfreeke) you are free to remove them from the list.